{
  "comments": [
    {
      "key": {
        "uuid": "9bc1fe7b_4e6e23ea",
        "filename": "kubernetes/dcaegen2/components/dcae-cloudify-manager/values.yaml",
        "patchSetId": 26
      },
      "lineNbr": 136,
      "author": {
        "id": 2319
      },
      "writtenOn": "2021-10-04T14:04:40Z",
      "side": 1,
      "message": "you\u0027re creating a \"patch\" user but don\u0027t use it no?",
      "revId": "9c8aa02672f1b3d8a1f799aed363a63ed38c626e",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "d649ff06_4787ee41",
        "filename": "kubernetes/dcaegen2/components/dcae-cloudify-manager/values.yaml",
        "patchSetId": 26
      },
      "lineNbr": 136,
      "author": {
        "id": 2627
      },
      "writtenOn": "2021-10-04T17:55:18Z",
      "side": 1,
      "message": "without this, cloudify generates error access forbidden trying to patch a configmap",
      "parentUuid": "9bc1fe7b_4e6e23ea",
      "revId": "9c8aa02672f1b3d8a1f799aed363a63ed38c626e",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "dec2b0e7_db224a4f",
        "filename": "kubernetes/dcaegen2/components/dcae-cloudify-manager/values.yaml",
        "patchSetId": 26
      },
      "lineNbr": 136,
      "author": {
        "id": 2627
      },
      "writtenOn": "2021-10-04T17:58:41Z",
      "side": 1,
      "message": "it seems this was already in the chart, i removed it now",
      "parentUuid": "d649ff06_4787ee41",
      "revId": "9c8aa02672f1b3d8a1f799aed363a63ed38c626e",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    }
  ]
}